### PR TITLE
Truncate crash functions

### PR DESCRIPTION
**Pull request by ernestask**
See commits.

This mainly truncates the various strings displayed in the web UI. Some visuals incoming as well.

### CODE REVIEW

**Comment by ernestask** [comment]
Can’t upload webums: https://cloud.gnome.org/s/EC68xxexDyZTB7c

**Comment by ernestask** [comment]
The previous attempt with horizontal limiting and ellipsization is too fiddly, given that we’re doing this all in a table, where we have to apply a bunch of styles, resulting in the crash function column width being controlled largely by the header, so expanding the text would yield a long and narrow one.

File: src/webfaf/static/js/custom.js [hunk 1]
```diff
@@ -172,6 +172,43 @@ $(document).ready(function() {
...
+        }
+        else if (showCollapser) {
+          expander.innerHTML = '[-]';
+        }
+
+        expander.hidden = !(showExpander || showCollapser);
+      }
+    });
+
+    document.querySelectorAll(".crash-fn-col").forEach(element => {
```
Comment (michalfabik) [inline src/webfaf/static/js/custom.js]: s/crash-fn-col/crash-fn-container/

File: src/webfaf/static/js/custom.js [hunk 2]
```diff
@@ -172,6 +172,43 @@ $(document).ready(function() {
...
+        const expanded = $(container).hasClass('expanded');
+        if (expanded && (container.scrollHeight == expander.scrollHeight)) {
+          container.classList.remove('expanded');
+
+          return;
+        }
+        const showExpander = !expanded && (container.scrollHeight > container.clientHeight);
+        const showCollapser = expanded;
+        if (showExpander) {
+          expander.innerHTML = '[+]';
```
Comment (michalfabik) [inline src/webfaf/static/js/custom.js]: Suggest using [show more] and [show less] instead. The [+] and [-] remind me of forums with Reddit-like layout where it's used to expand/collapse a subtree of posts/messages and it looks a little odd centered at the bottom like this. But maybe it's just me.

File: src/webfaf/static/js/custom.js (hunk 1 above)
Comment (ernestask) [reply src/webfaf/static/js/custom.js]: Have you tried it with that?

Comment (ernestask) [reply src/webfaf/static/js/custom.js]: Oh, I did drop the class altogether.

File: src/webfaf/static/js/custom.js (hunk 2 above)
Comment (ernestask) [reply src/webfaf/static/js/custom.js]: Looking for analogies here is a bit pointless, but I can make it textual, sure. Should be fine, given that we don’t localize strings.

File: src/webfaf/templates/reports/item.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
 {% block title -%}
   {{ "Report #%d - %s" | format(report.id, component.name) -}}
-  {{ " in %s" | format(report.backtraces[0].crash_function) if report.backtraces | length > 0 }}
+  {{ " in %s" | format(report.backtraces[0].crash_function) | truncate(80, True) if report.backtraces | length > 0) }}
```
Comment (mkutlak) [inline src/webfaf/templates/reports/item.html]: unexpected ')'

**Pull request merged**